server/base/src/main/java/org/apache/accumulo/server/util/RpcWrapper.java

```diff
@@ -36,20 +42,51 @@
  * @since 1.6.1
  */
 public class RpcWrapper {
+  private static final Logger log = LoggerFactory.getLogger(RpcWrapper.class);
+
+  public static <T> T service(final T instance, @SuppressWarnings("rawtypes") final Map<String,ProcessFunction<T,? extends TBase>> processorView) {
+    // Get a handle on the isOnewayMethod and make it accessible
+    final Method isOnewayMethod;
+    try {
+      isOnewayMethod = ProcessFunction.class.getDeclaredMethod("isOneway");
+    } catch (NoSuchMethodException e) {
+      throw new RuntimeException("Could not access isOneway method", e);
+    } catch (SecurityException e) {
+      throw new RuntimeException("Could not access isOneway method", e);
+    }
+    isOnewayMethod.setAccessible(true);
```

Comment:
No, we need it to be accessible to invoke the method later, I believe. I will double check the API -- I don't use reflection enough to remember this all for certain.